src/ray/object_manager/object_manager.cc

```diff
@@ -14,7 +14,7 @@ ObjectManager::ObjectManager(asio::io_service &main_service,
     : client_id_(gcs_client->client_table().GetLocalClientId()),
       object_directory_(new ObjectDirectory(gcs_client)),
       store_notification_(main_service, config.store_socket_name),
-      store_pool_(config.store_socket_name),
+      buffer_pool_(config.store_socket_name, config.object_chunk_size, 2*config.max_sends),
```

Comment:
Add a comment explaining the rationale for `2*config.max_sends`. Style guide also recommends adding a /*release_delay=*/ comment right before the argument value for parameters whose purpose isn't clear. Also, I'm a little surprised linting didn't catch this, but add spaces around the `*`.